src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/TestData/Expected/Hl7v2/ORU_R01/ORU-R01-RMGEAD-expected.json

```diff
@@ -65,10 +80,14 @@
               ]
             }
           },
-          { "value": "845439" },
-          { "value": "1045813" }
+          {
+            "value": "845439"
```

Comment:
A value without system? Are we missing something?

Reply:
This issue is caused by the logics in `DiagnosticReport`. The generated `code`, `system` and `value` are wrongly placed. Each value should have a `coding` accompanied as the one in Line 75. @ACMoretxj and I will look into this together.

Reply:
(The issues exist in both Handlbars templates and DotLiquid templates, as the generation logics are very similar except syntax differences.)

Reply:
Agree with @BoyaWu10 , there are also many same issues, and it might be fixed by updating the mapping table. I'll write this into our template improving plan.